public/docs/vue.html

```diff
@@ -60,9 +62,98 @@ <h2>table-sort-js</h2>
         </div>
       </div>
       <div class="main_content">
-        <div class="header"></div>
+        <div class="header">
+          Example of how to use table-sort.js in a Vue application. You need to have Vue CLI or Webpack (with Vue Loader). For more information on Vue CLI and/or Webpack, please see their respective documentation! The code below should get things working..
+        </div>
+        <div class="info" style="text-align:left">
+          Download with NPM:
+          <code
+            style="background-color: #839496; color: white; padding-top: 3px"
+            >npm install table-sort-js -g</code
+          >
+          <br>
+        </div>
+        <li style="text-align: center">
+          Insert this code block into your entry point JavaScript file like main.js
+        </li>
+        <pre><code style="font-size:1rem" data-language="js">
+          import Vue from 'vue'
+          import App from './App'
+          import tableSort from 'table-sort-js/table-sort.js';
+
+          Vue.config.productionTip = false
+          Vue.use(tableSort)
+
+
+          new Vue({
+            render: h => h(App),
+          }).$mount('#app')
+        </code>
+        </pre>
+        <li style="text-align: center">
+          Insert this markup into your html
+        </li>
+        <pre><code style="font-size:1rem; margin-bottom:10px" data-language="html">
+          &lt;!DOCTYPE html&gt;
+          &lt;html&gt;
+
+          &lt;head&gt;
+            &lt;meta charset="utf-8"&gt;
+            &lt;meta name="viewport" content="width=device-width,initial-scale=1.0"&gt;
+            &lt;title>Table-Sort-JS&lt;/title&gt;
+          &lt;/head&gt;
+
+          &lt;body&gt;
+            &lt;div id="app"&gt;&lt;/div&gt;
+          &lt;/body&gt;
+
+          &lt;/html&gt;
+
+</code></pre>
```

Comment:
Not sure if we need to say this for HTML :thinking: , it's not really unique to table-sort-js. index.html has `<div id ="app">` by default anyway; so they don't need to insert the markup into their HTML.